analysis/druidrestoration/src/modules/features/RegrowthAndClearcasting.tsx

```diff
@@ -220,6 +235,30 @@ class RegrowthAndClearcasting extends Analyzer {
     );
   }
 
+  get guideTimeline() {
+    const values = this.castRegrowthLog.map((rgCast) => {
+      let message = '';
+      if (rgCast.reason === 'innervate') {
+        message = 'Free due to Innervate';
+      } else if (rgCast.reason === 'ns') {
+        message = "Free due to Nature's Swiftness";
+      } else if (rgCast.reason === 'clearcast') {
+        message = 'Free due to Clearcasting';
+      } else if (rgCast.reason === 'abundance') {
+        message = `Cheap due to ${rgCast.abundanceStacks} stacks of Abundance`;
+      } else if (rgCast.reason === 'triage') {
+        message = 'Cast full price on a low health target';
+      } else if (rgCast.reason === 'bad') {
+        message = 'Cast full price on a high health target';
```

Comment:
showing the target name here would be helpful for quickly understanding the cast (e.g. if it shows your tank's name, then that is different than maybe showing yourself or another healer)

Reply:
I considered adding more info, like the target health / name / overheal percent, but decided to not overload it. I'll take another look - also is there an easy way to get target name from targetID?